docs-source/content/faq/volumes.md

```diff
@@ -0,0 +1,49 @@
+> I need to provide an instance with access to a Persistent Volume Claim
+
+Some applications need access to a file, either to read data or to provide additional logging beyond what is 
+built into the operator. One common way of doing that within Kubernetes is to create a 
+[PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC) and map
+it to a file. The domain configuration can then be used to provide access to the claim across the domain, 
+within a single cluster, or for a single server.
+In each case, the access is configured within the ``ServerPod`` element of the configuration. For example, here is
```

Comment:
"In each case, the access is configured within the ``ServerPod`` element of the configuration."

looks incorrect.  (1) Element name is 'serverPod' not 'ServerPod'., (2) The cluster configuration is in 'clusters', not 'ServerPod', as per your example below.

Reply:
(1) Yes, it should be 'serverPod'
(2) I don't understand your second comment. The 'volumes' and 'volumeMounts' fields are under serverPod which is inherited by Cluster from BaseConfiguration.

Reply:
You can configure either the admin server or a managed server. The latter is configured under the managedServers: array. I am assuming that the users know _something_ about the domain resource.

Reply:
I don't think the inheritance hierarchy is obvious to users - if one is directly working with domain resource XML there's nothing there to tell you that 'clusters' inherits the same type as 'serverPod'.  It's much more clear IMO just to directly state the names of the parent stanzas that support volume and volumeMount child stanzas.  MBeans in WL work very similarly - two beans may inherit the same fields, but we typically treat them independently in documentation.

Reply:
> Did you disagree that PersistentVolumeClaim should be in code font?

No, just missed it.